_examples/basic/2-realtime-feed/consumer/main.go

```diff
@@ -22,72 +22,72 @@ var (
 )
 
 func main() {
-	logger := watermill.NewStdLogger(true, true)
+	logger := watermill.NewStdLogger(false, false)
+	logger.Info("Starting the consumer", watermill.LogFields{})
+
 	pub, err := kafka.NewPublisher(brokers, marshaler, nil, logger)
 	if err != nil {
 		panic(err)
 	}
 
-	r, err := message.NewRouter(
-		message.RouterConfig{},
-		logger,
-	)
+	r, err := message.NewRouter(message.RouterConfig{}, logger)
 	if err != nil {
 		panic(err)
 	}
 
-	retryMiddleware := middleware.Retry{}
-	retryMiddleware.MaxRetries = 1
-	retryMiddleware.InitialInterval = time.Millisecond * 10
+	retryMiddleware := middleware.Retry{
+		MaxRetries:      1,
+		InitialInterval: time.Millisecond * 10,
+	}
 
 	poisonQueue, err := middleware.PoisonQueue(pub, "poison_queue")
 	if err != nil {
 		panic(err)
 	}
 
 	r.AddMiddleware(
-		// limiting processed messages to 10 per second
-		middleware.NewThrottle(100, time.Second).Middleware,
+		// Limit incoming messages to 10 per second
+		middleware.NewThrottle(10, time.Second).Middleware,
 
-		// retry middleware retries message processing if error occurred in handler
-		poisonQueue,
-
-		// if retries limit was exceeded, message is sent to poison queue (poison_queue topic)
+		// Retry middleware retries message processing if an error occurred in the handler
 		retryMiddleware.Middleware,
 
-		// recovered recovers panic from handlers
+		// If the retries limit is exceeded, the message is sent to the poison queue (poison_queue topic)
+		poisonQueue,
+
+		// Recoverer middleware recovers panic from handlers
 		middleware.Recoverer,
```

Comment:
basically, it should be on top in case of above middlewares failure

Reply:
Ah right, I've changed this because the comments were hard to read that way. I'll update it somehow.

Reply:
Ah no, I changed the poison queue and retry order. But I guess this is wrong as well.

Reply:
See 3b1f8a17e4013c7a05e5224fa4653a845de7c7fe